web-client/src/app/views/register/register.page.ts

```diff
@@ -1,31 +1,68 @@
 import { Component, OnInit } from '@angular/core';
-import { FormBuilder, FormGroup } from '@angular/forms';
+import { AbstractControl, FormBuilder, FormGroup, Validators } from '@angular/forms';
 
 @Component({
   selector: 'app-register',
   templateUrl: './register.page.html',
   styleUrls: ['./register.page.scss'],
 })
 export class RegisterPage implements OnInit {
-  registrationForm: FormGroup;
+  public registrationForm: FormGroup;
 
+  
   constructor(private formBuilder: FormBuilder) {
     this.initializeForm();
   }
-
+  
   ngOnInit() {}
-
-  initializeForm(): void{
+  
+  initializeForm(): void {
     this.registrationForm = this.formBuilder.group({
-      firstName: '',
-      lastName: '',
-      mobile: '',
-      password: '',
-      confirmPassword: ''
-    })
-  };
-
+      firstName: ['', Validators.compose([Validators.required])],
+      lastName: ['', Validators.compose([Validators.required])],
+      mobile: [
+        '',
+        Validators.compose([
+          Validators.required,
+          Validators.minLength(1),
```

Comment:
I think minLength here should be 10

Reply:
Min length is 4 and max is 10 from what we discussed.